interfaces/Portalicious/src/main.ts

```diff
@@ -1,5 +1,10 @@
 /// <reference types="@angular/localize" />
 
+import '@angular/common/locales/global/ar';
+import '@angular/common/locales/global/en';
+import '@angular/common/locales/global/es';
+import '@angular/common/locales/global/fr';
+import '@angular/common/locales/global/nl';
```

Comment:
I've included all of these already, so enable the formatting of dates/numbers in all the locales. (Not really tested, but I think it works the same as in the old Portal)